### PR TITLE
Implement the API endpoint for creating a new release

### PR DESCRIPTION
**Pull request by AvdLee**
Implements the [API endpoint to create a new release](https://developer.github.com/v3/repos/releases/#create-a-release) on a GitHub repository.

### CODE REVIEW

**Comment by AvdLee** [comment]
@pietbrauer both PRs are back up to date with master 👍

File: OctoKit/Releases.swift [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    ///   - repo: The repository on which the release needs to be created.
+    ///   - tagName: The name of the tag.
+    ///   - targetCommitish: Specifies the commitish value that determines where the Git tag is created from. Can be any branch or commit SHA. Unused if the Git tag already exists. Default: the repository's default branch (usually master).
+    ///   - name: The name of the release.
+    ///   - body: Text describing the contents of the tag.
+    ///   - prerelease: `true` to create a draft (unpublished) release, `false` to create a published one. Default: `false`.
+    ///   - draft: `true` to identify the release as a prerelease. `false` to identify the release as a full release. Default: `false`.
+    ///   - completion: Callback for the outcome of the created release.
+    @discardableResult
+    func postRelease(_ session: RequestKitURLSession = URLSession.shared, owner: String, repository: String, tagName: String, targetCommitish: String, name: String, body: String, prerelease: Bool = false, draft: Bool = false, completion: @escaping (_ response: Response<Release>) -> Void) -> URLSessionDataTaskProtocol? {
```
Comment (pietbrauer) [inline OctoKit/Releases.swift]: As I see it only `tagName` (except from `owner` and `repository`) is a required field.

https://developer.github.com/v3/repos/releases/#input

Comment (AvdLee) [reply OctoKit/Releases.swift]: Good catch! I'll update accordingly

**Comment by pietbrauer** [comment]
Thanks a lot!

**Pull request merged**